src/components/AvatarGroup/AvatarGroup.st.css

```diff
@@ -0,0 +1,125 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, fallback;
+}
+
+:import {
+    -st-from: "wix-ui-core/index.st.css";
+    -st-named: Avatar;
+}
+
+:vars {
+    /*Defaults*/
+    DefaultAvatarBorderColor: color-1;
+}
+
+:vars {
+    /*Overrides*/
+     AvatarBorderColor: --overridable;
+}
+
+:vars {
+    avatarBorderColorValue: fallback(value(AvatarBorderColor), value(DefaultAvatarBorderColor));
+}
+
+.root {
+    -st-states: size(enum(large, medium, small, xSmall, xxSmall));
+    display: inline-flex;
+    flex-direction: row;
+}
+
+.root:size(large) {
+    padding-left: 14px;
+}
+
+.root:size(medium) {
+    padding-left: 12px;
+}
+
+.root:size(small) {
+    padding-left: 10px;
+}
+
+.root:size(xSmall) {
+    padding-left: 6px;
+}
+
+.root:size(xxSmall) {
+    padding-left: 4px;
+}
+
+.root .avatarContainer {
+    overflow: hidden;
+    position: relative;
+}
+
+.root:size(large) .avatarContainer {
+    margin-left: -14px;
+}
+
+.root:size(medium) .avatarContainer {
+    margin-left: -12px;
+}
+
+.root:size(small) .avatarContainer {
+    margin-left: -10px;
+}
+
+.root:size(xSmall) .avatarContainer {
+    margin-left: -6px;
+}
+
+.root:size(xxSmall) .avatarContainer {
+    margin-left: -4px;
+}
+
+.avatar {
+    -st-extends: Avatar;
+}
+
+.root .avatar::content {
+    border: solid 1px;
+    border-color: color(value(avatarBorderColorValue));
+    border-radius: 50%;
```

Comment:
Sorry for the picky comment 😬 
But lets move the `border-radius` to `Avatar.st.css`.

Reply:
Done